IPs/RSKIP197.md

```diff
@@ -0,0 +1,78 @@
+# Error Handling for Precompiled Contracts
+
+|RSKIP          |197           |
+| :------------ |:-------------|
+|**Title**      |Error Handling for Precompiled Contracts |
+|**Created**    |15-12-2020 |
+|**Author**     |FJ |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Accepted |
+
+## Abstract
+
+This RSKIP specifies a new expected behavior for a precompiled contract call: any expected failed call pushes a zero into the stack and reverts the state before the execution.
+
+## Motivation
+
+Provides a mechanisim to handle errors for precompiled calls by checking the returned state. Before Iris hardfork the transaction wasn't reverted and consumed all the remaining gas from the entire call.
```

Comment:
-> Before the iris hard-fork, a precompiled contract could not revert, but only rise an OOG, consuming all gas passed to the CALL.

Reply:
I've left this as motivation